state/modelgeneration.go

```diff
@@ -424,6 +422,52 @@ func (g *Generation) Commit(userName string) (int, error) {
 	return newGenId, nil
 }
 
+// assignedWithAllUnits generates a new value for the branch's
+// AssignedUnits field, to indicate the all units of changed applications
```

Comment:
Gah, "that".

Reply:
I don't really understand why.

Reply:
This is for the history tracking requirement. The generation will then indicate all the units that were extant at the time of commit.